compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -1608,6 +1608,22 @@ loco::NodeShape infer_unpack(const luci::CircleUnpack *node)
   return loco::NodeShape{output_shape};
 }
 
+loco::NodeShape infer_unidirectionalsequencelstm(const luci::CircleUnidirectionalSequenceLSTM *node)
+{
+  auto input_shape = loco::shape_get(node->input()).as<loco::TensorShape>();
+  auto recurrent_to_output_weights =
+      loco::shape_get(node->recurrent_to_output_weights()).as<loco::TensorShape>();
+  auto rank = static_cast<int32_t>(input_shape.rank());
```

Comment:
Is there any reason of this `static_cast`?

Reply:
`input_shape.rank()` is a type of `uint32_t`. so, I just use static_cast.

Reply:
@struss got it. But it might as well goes on as it is without casting because rank shouldn't be minus value, which is why most of `rank` methods returns `uint32_t` I think.

Reply:
Oh, I will update without casting. thanks for pointing out!